### DISCUSSION TITLE
关于使用hook方法，只能返回alova实例的问题

### DISCUSSION THREAD

**Discussion by x007xyz** [alovajs/alova#261] [Q&A]
### 请详细描述bug

```
const {
    page: [page, setPage],
    data,
    loading,
  } = usePagination((page, pageSize) => location && fetchPlaceByText({ page, pageSize, keywords: inputText, location }), {
    initialData: {
      total: 0,
      data: [],
    },
    total: (response) => response.count,
    data: (response) => response.pois,
    initialPage: 1,
    initialPageSize: 20,
    append: true,
    immediate: false,
    watchingStates: [inputText, location],
  })
```
在上面的代码中，我需要监听inputText, location而且发送请求，但是location如果为空，其实我并不希望发送请求，所以在hook中location为空，没有调用alova实例方法；但是现在和预想的结果不同，在location为空时，直接报错了
![image](https://github.com/alovajs/alova/assets/13807549/7a2b7921-2a03-4b07-81d7-0e8892a51338)
在这种场景下该如何使用才能实现我想要的效果呢？

### 复现步骤

根据描述场景就可以触发错误了

### alova 版本

2.15.0

**Comment by JOU-amjs** [comment]
问题类的请在Discussion中提问。
你的问题在`location && fetchPlaceByText(...)`，当location为`falsy`值时请求内会接收到空值，因此报了这个错。

**Comment by MeetinaXD** [comment]
回答得有些晚，但以防以后有人需要这个答案。
Late a little bit, but in case someone else needs it.

usePagination 是使用 useWatcher 实现的，因此你可以使用 `useWatcher` 中提供的选项 `sendable`。
usePagination uses useWatcher, so you can use the option `sendable` provided in `useWatcher`.

一个示例如下：
Example:
<img width="431" alt="image" src="https://github.com/alovajs/alova/assets/40497962/ec7b633b-b115-469a-a0c0-00d844d30c27">
